sys/include/net/ng_netopt.h

```diff
@@ -137,6 +137,11 @@ typedef enum {
     NG_NETOPT_TX_END_IRQ,
     NG_NETOPT_AUTOCCA,              /**< en/disable to check automatically
                                      *   before sending the channel is clear. */
+
+
+    NG_NETOPT_ACCESS_ADDRESS,     /**< get/set BLE access address */
```

Comment:
I'll probably be adding more netopts so I think it makes sense to add BLE to the defines. Would that make them get too long though?